tests/test_sort_query.py

```diff
@@ -249,6 +249,7 @@ class TextItem(Base):
             type = sa.Column(sa.Unicode(255))
 
             __mapper_args__ = {
+                'polymorphic_identity': u'text_item',
```

Comment:
Copy/paste :)

Reply:
I guess this repo would benefit from pyupgrade being run over it, but maybe not in this PR since it would obscure already singnificant changes made here

Reply:
> I guess this repo would benefit from pyupgrade being run over it, but maybe not in this PR since it would obscure already significant changes made here.

Agreed. I've removed all the `u` in this file.